### PR TITLE
Add Saturday plan back into Paper product

### PR DESCRIPTION
**Pull request by rupertbates**
## What are you doing in this PR?

[**Trello Card**](https://trello.com)

## Is this an AB test?
- [ ] Yes
- [ ] No

If this is an AB test, PR reviewers should open and check the Optimize test.
[**Optimize Link**](https://optimize.google.com/optimize/home)

## Accessibility test checklist
 - [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
 - [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
 - [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### CODE REVIEW

File: support-frontend/assets/components/product/productOption.jsx [hunk 1]
```diff
@@ -95,34 +182,48 @@ function ProductOption(props: Product) {
...
         <ThemeProvider theme={buttonReaderRevenue}>
           <LinkButton
+            css={button}
             href={props.href}
             onClick={props.onClick}
-            aria-label={`${props.title}- ${props.buttonCopy}`}
+            aria-label={`${props.title}- ${props.buttonCopy} now`}
           >
-            {props.buttonCopy}
+            {props.buttonCopy}<span css={buttonCopySpan}>&nbsp;now</span>
```
Comment (GHaberis) [inline support-frontend/assets/components/product/productOption.jsx:226]: Is this `<span>` appending "now" to the button copy necessary? I think it'd be easier to find where the button copy string is defined, before it's passed down in the props, and to just add "now" to the button copy string. Plus, this is a shared Component, so hard coding "now" on the end here can have unintended side affects. EG. The Guardian Weekly CTA on the pricing cards becomes:

Screenshot 2021-09-20 at 09 08 20

File: support-frontend/assets/components/packshots/paper-packshot.jsx [hunk 2]
```diff
@@ -5,31 +5,13 @@ import GridImage from 'components/gridImage/gridImage';
...
-      srcSizes={[756, 500, 140]}
-      sizes="(max-width: 740px) 100%,
-             (max-width: 980px) 204px,
-             (max-width: 1140px) 227px,
-             252px"
+      srcSizes={[500, 140]}
+      sizes="(max-width: 739px) 140px,
+             (max-width: 979px) 500px,
+             (max-width: 1140px) 500px,
+             500px"
```
Comment (GHaberis) [inline support-frontend/assets/components/packshots/paper-packshot.jsx:14]: Is the 1px difference in some of the media-queries here necessary?

File: support-frontend/assets/components/product/productOption.jsx (hunk 1 above)
Comment (marialani) [reply support-frontend/assets/components/product/productOption.jsx:226]: Ah yes I'd missed where it was shared. With the latest design the text for the button now changes for different screen sizes so had opted to conditionally include the "now" copy using media queries but will change this now.

File: support-frontend/assets/components/packshots/paper-packshot.jsx (hunk 2 above)
Comment (marialani) [reply support-frontend/assets/components/packshots/paper-packshot.jsx:14]: Here I'd followed the convention in the codebase but I do believe the 1px difference may not be necessary due to cascading

Comment (GHaberis) [reply support-frontend/assets/components/packshots/paper-packshot.jsx:14]: OK great, if the previous implementation went against the convention then we should retain the changes you've made.

File: support-frontend/assets/components/product/productOption.jsx (hunk 1 above)
Comment (GHaberis) [reply support-frontend/assets/components/product/productOption.jsx:226]: OK I see, good to know! I'd question whether the inclusion of "now" is really essential for the objective of this piece of work - to launch this Newspaper campaign, I'd say the current wording on the CTA without the "now" is fine, and if we do want to implement different wording at different breakpoints we create a separate card/PR to do this.

**Review by GHaberis — APPROVED** [review]
👍

**Pull request merged**